aiocouch/document.py

```diff
@@ -272,6 +305,18 @@ def __init__(self, database: "database.Database"):
         super().__init__(database, "_security")
         del self._data["_id"]
 
+    async def __aenter__(self) -> "SecurityDocument":
```

Comment:
I guess this overload is needed for mypy. I wonder if there's a more elegant solution using `typing.cast` or something?

Reply:
We can omit the `__aenter__` here; mypy won't complain, but we'll lose the IDE's auto-complete support for Security Document's methods (at least VS Code when I tested it).

Thanks for the `typing.cast` advice. Here's another approach I came up with. Is this okay to use?
```diff
async def __aenter__(self) -> "SecurityDocument":
-        await self.fetch(discard_changes=True)
-        return self
+        return cast(SecurityDocument, await super().__aenter__())
```

Reply:
We can omit the `__aenter__` here; mypy won't complain, but we'll lose the IDE's auto-complete support for Security Document's methods (at least VS Code didn't show them when I tested this omission).

Thanks for the `typing.cast` advice. Here's another approach I came up with. Is this okay to use?
```diff
async def __aenter__(self) -> "SecurityDocument":
-        await self.fetch(discard_changes=True)
-        return self
+        return cast(SecurityDocument, await super().__aenter__())
```

Reply:
Yeah, that seems like what I had in mind